quesma/model/metrics_aggregations/percentile_ranks.go

```diff
@@ -11,14 +11,15 @@ import (
 )
 
 type PercentileRanks struct {
-	ctx context.Context
+	ctx         context.Context
+	percentiles []string
```

Comment:
I would add comment or even better name of that value. Not clear what it is.
```suggestion
	cutValues []string // count(if(field<$cutValue, 1, NULL))/count(*)*100
```